auacm/app/database.py

```diff
@@ -0,0 +1,11 @@
+'''Database handlers.'''
+from sqlalchemy import create_engine
+from sqlalchemy.orm import Session
+from sqlalchemy.ext.automap import automap_base
+
+# database setup
+Base = automap_base()
+engine = create_engine('mysql://acm@localhost/acm')
```

Comment:
I think pulling away the account dependency is a good idea, but I'm not entirely sure that this is the right way to go about it yet.  Because config.py isn't checked into the repo, we need another way of providing default options.  I think the ideal approach would be to add this to config.py, but to also add this to setup scripts that can configure the default option.